mlflow/gateway/providers/base.py

```diff
@@ -14,6 +15,7 @@ class BaseProvider(ABC):
 
     NAME: str = ""
     SUPPORTED_ROUTE_TYPES: Tuple[str, ...]
+    CONFIG_TYPE: Type[ConfigModel] = None
```

Comment:
```suggestion
    CONFIG_TYPE: Type[ConfigModel]
```

Do we need `= None`?

Reply:
removed it